stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveDataBenchmark.kt

```diff
@@ -65,7 +65,7 @@ internal class MessageListItemLiveDataBenchmark {
             }
         }
         println("changing typing information 100 times on a message list with ${items.size} items took $duration milliseconds")
-        Truth.assertThat(duration).isLessThan(20)
```

Comment:
Why this change?

Reply:
This test was flaky, sometimes it failed because the operations it measured took 21ms instead of 20ms, this makes it more tolerant